### PR TITLE
misc(asset-saver): tweak output format of logAssets

### PR DESCRIPTION
**Pull request by paulirish**
For slightly cleaner log retrieval from lightrider

### CODE REVIEW

File: lighthouse-core/lib/asset-saver.js [hunk 1]
```diff
@@ -326,13 +326,16 @@ async function saveAssets(artifacts, audits, pathWithBasename) {
...
+    // eslint-disable-next-line no-console
+    console.log(`loggedAsset %%% devtoolslog-${passAssets.passName}.json %%% ${dtlogdata}`);
     const traceIter = traceJsonGenerator(passAssets.traceData);
     let traceJson = '';
     for (const trace of traceIter) {
       traceJson += trace;
     }
-    log.log(`trace-${passAssets.passName}.json`, traceJson);
+    // eslint-disable-next-line no-console
+    console.log(`loggedAsset %%% trace-${passAssets.passName}.json %%% ${traceJson}`);
```
Comment (patrickhulce) [inline lighthouse-core/lib/asset-saver.js]: is there still the max `console.log` length problem or was that fixed?

File: lighthouse-core/lib/asset-saver.js [hunk 2]
```diff
@@ -326,13 +326,16 @@ async function saveAssets(artifacts, audits, pathWithBasename) {
 async function logAssets(artifacts, audits) {
   const allAssets = await prepareAssets(artifacts, audits);
   allAssets.map(passAssets => {
-    log.log(`devtoolslog-${passAssets.passName}.json`, JSON.stringify(passAssets.devtoolsLog));
+    const dtlogdata=JSON.stringify(passAssets.devtoolsLog);
```
Comment (patrickhulce) [inline lighthouse-core/lib/asset-saver.js]: spaces?

File: lighthouse-core/lib/asset-saver.js (hunk 1 above)
Comment (paulirish) [reply lighthouse-core/lib/asset-saver.js]: fixed. :) yay

File: lighthouse-core/lib/asset-saver.js (hunk 2 above)
Comment (paulirish) [reply lighthouse-core/lib/asset-saver.js]: nah it's more straightforward without any extra whitespace

Comment (brendankenny) [reply lighthouse-core/lib/asset-saver.js]: > nah it's more straightforward without any extra whitespace

uh...wha?

Comment (paulirish) [reply lighthouse-core/lib/asset-saver.js]: OHHH i thought the suggestion was to add a spaceindent arg to stringify. :)  🤡

**Pull request merged**